.dockerignore

```diff
@@ -1,5 +1,2 @@
 # All env files
-*.env
-*.env.template
-*.env.test
```

Comment:
We would probably choose to allow the `.env.test` file into the container, but we still definitely want to avoid our actual credentials being baked in - i.e. having `.env` here is still important!